packages/aws-cdk-lib/aws-ecs-patterns/README.md

```diff
@@ -888,6 +888,22 @@ const applicationLoadBalancedFargateService = new ecsPatterns.ApplicationLoadBal
 });
 ```
 
+### Customize Container Name for ScheduledFargateTask
```

Comment:
We should also add the readme to scheduled ecs task too

Reply:
done